### DISCUSSION TITLE
How can Change enjine between Chatgpt 4 and chatgpt 3.5

### DISCUSSION THREAD

**Discussion by alirezach** [lifan0127/ai-research-assistant#56] [General]
Hi,
I haven't ChatGPT 4 Api and I can't find settings for this

**Comment by lifan0127** [comment]
Hi @alirezach , GPT-3.5-Turbo is no longer supported as it often struggles to produce the correct output necessary for this plugin. 

Please check out this OpenAI support page for GPT-4 access: https://help.openai.com/en/articles/7102672-how-can-i-access-gpt-4
